src/components/views/spaces/threads-activity-centre/ThreadsActivityCentre.tsx

```diff
@@ -65,41 +68,55 @@ export function ThreadsActivityCentre({ displayButtonLabel }: ThreadsActivityCen
                 }
             }}
         >
-            <Menu
-                align="end"
-                open={open}
-                onOpenChange={(newOpen) => {
-                    // Track only when the Threads Activity Centre is opened
-                    if (newOpen) PosthogTrackers.trackInteraction("WebThreadsActivityCentreButton");
-
-                    setOpen(newOpen);
-                }}
-                side="right"
-                title={_t("threads_activity_centre|header")}
-                trigger={
+            {releaseAnnouncementEnabled ? (
+                <ReleaseAnnouncement
```

Comment:
This check looks redundant to the usage of `useIsReleaseAnnouncementEnabled` inside the ReleaseAnnouncement component

Reply:
Yes and not, I want to avoid to render the compound menu and the RA on the same button

Reply:
Fair enough